cmd/update/cluster/command.go

```diff
@@ -26,6 +26,7 @@ Options:
   --name <cluster-name>             	Name of the cluster to update.
   --namespace <cluster-namespace>   	Namespace of the cluster.
   --release-version <release-version>   Update the cluster to a release version. The release version must be higher than the current release version.
+  --remove-schedule				Optionally: Remove the scheduled time from the cluster.
```

Comment:
there seems to be a spacing/alignment error before "Optionally"

Reply:
thanks ❤️, fixed